spec/support/rails_template_with_data.rb

```diff
@@ -35,6 +156,137 @@
   scope :my_posts do |posts|
     posts.where(author_id: current_admin_user.id)
   end
+
+  controller do
+
+    def scoped_collection
+      super.includes(:author, :category, :taggings)
```

Comment:
@javierjulio  it can be simplified using includes on resource level. take a look 
https://github.com/activeadmin/activeadmin/pull/3464#issue-44728695